services/movie.py

```diff
@@ -0,0 +1,30 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
+               ) -> list[Movie]:

```

Comment:
If argument can be None, you must show it in type hint. Fix everywhere, please
```suggestion
def get_movies(genres_ids: list[int] | None = None,
               actors_ids: list[int] | None = None
               ) -> list[Movie]:
```

Reply:
def get_movies_sessions(session_date: datetime | None = None
E   TypeError: unsupported operand type(s) for |: 'module' and 'NoneType'

But does not work for datetime?